src/lib.rs

```diff
@@ -97,6 +97,7 @@ use alloc::vec::Vec;
 use core::cmp::{max, min};
 use core::iter::repeat;
 use core::ops::Range;
+use core::usize;
```

Comment:
You shouldn't need to import `usize`

Reply:
I removed it, I am not sure why it is added and is it hurt to add it ?

Reply:
There should be no harm in adding it, but it is unnecessary and bad code style to import things you don't need to import.